packages/neuron-ui/src/components/SUDTSend/index.tsx

```diff
@@ -386,35 +399,42 @@ const SUDTSend = () => {
                       disabled={sendState.sendAll}
                       error={errors[field.key]}
                       className={styles[field.key]}
+                      placeholder={field.placeholder}
                     />
 
                     {field.key === Fields.Address && options?.length ? (
                       <>
-                        <RadioGroup
-                          defaultValue={sendType}
-                          onChange={onChangeSendType}
-                          itemClassName={styles.optionItem}
-                          options={options.map(item => ({
-                            value: item.key,
-                            label: t(`s-udt.send.${item.label}`, item?.params),
-                            suffix: item.tooltip ? (
-                              <div className={styles.tipItem}>
-                                <Tooltip
-                                  tip={
-                                    <p className={styles.tooltip}>{t(`s-udt.send.${item.tooltip}`, item?.params)}</p>
-                                  }
-                                  showTriangle
-                                >
-                                  <AttentionOutline className={styles.attention} />
-                                </Tooltip>
-                              </div>
-                            ) : null,
-                            tip:
-                              item.key === SendType.secp256Cheque && !isMainnet ? (
-                                <div className={styles.selectError}>{t('messages.light-client-cheque-warning')}</div>
+                        {options.length > 1 ? (
```

Comment:
IMO, if `options`.length is 1, we can also use `RadioGroup`.

Reply:
The single option needs to support unchecking, so it need to use the checkbox